apps/dashboard/src/components/buttons/MismatchButton.tsx

```diff
@@ -254,9 +262,8 @@ const NoFundsNotice: React.FC<NoFundsNoticeProps> = ({ symbol }) => {
       });
       if (chainInfo.chainId === localhost.id) {
         await sdk.wallet.requestFunds(10);
-      } else if (chainInfo?.faucets && chainInfo.faucets.length > 0) {
-        const faucet = chainInfo.faucets[0];
-        window.open(faucet, "_blank");
+      } else if (chainInfo?.testnet) {
+        router.push(`/${chainInfo.chainId}/faucet`);
```

Comment:
FYI @MananTank im redirecting to chain/faucet for now, but with your change we can prob just redirect to the chain page. make sure we handle redirects or change this value